### PR TITLE
Update raiseDeps and GithubRepos with new Repos

### PR DESCRIPTION
**Pull request by ivy-edp**
As we discussed earlier, I have now added the files with the new repos.

### CODE REVIEW

**Comment by ivy-edp** [comment]
Are these the only locations or is there another file that I need to adapt? Should someone else look over it to make sure we did not forgot a repo?

**Review by ivy-lli — APPROVED** [review]
I don't have another repo in mind...
@ivy-rew, @alexsuter, @ivy-cst maybe you have another repo which is missing here?

File: github-repo-manager/src/main/java/com/axonivy/github/GitHubRepos.java [hunk 1]
```diff
@@ -41,6 +41,9 @@ public static List<String> repos(String version) {
           "process-editor-client",
           "process-editor-core",
           "inscription-client",
+          "config-editor-client",
+          "form-editor-client",
+          "ui-components",
```
Comment (alexsuter) [inline github-repo-manager/src/main/java/com/axonivy/github/GitHubRepos.java:46]: @ivy-edp is the new variable editor also covered? is this the config-editor-client?

**Comment by alexsuter** [comment]
Also update this file here:
build/raise-version/raise.sh

**Comment by alexsuter** [comment]
@ivy-edp Can you add a new section in README.md that probably these 3 files must be updated when introducing a new repo? Thanks.

File: github-repo-manager/src/main/java/com/axonivy/github/GitHubRepos.java (hunk 1 above)
Comment (ivy-lli) [reply github-repo-manager/src/main/java/com/axonivy/github/GitHubRepos.java:46]: yes the variable editor is in the config-editor-client repo

**Comment by ivy-lli** [comment]
> Also update this file here: build/raise-version/raise.sh

This looks up do date 👍

File: github-repo-manager/src/main/java/com/axonivy/github/GitHubRepos.java (hunk 1 above)
Comment (ivy-edp) [reply github-repo-manager/src/main/java/com/axonivy/github/GitHubRepos.java:46]: @alexsuter Yes, as far as i can tell and see, the variable editor lives in the config-editor-client repo and is therefore covered. @ivy-lli right?

**Review by alexsuter — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -7,3 +7,12 @@ Managing settings of GitHub repositories.
...
 and run the java application.
+
+## New GitHub repositories
+
+If the GitHub Repo Manager need to include 
+a new repository, the following three files 
+need to be updated manually:
+- `github-repo-manager/src/main/java/com/axonivy/github/GitHubRepos.java`
+- `build/raise-deps/raise.sh`
+- `build/raise-version/raise.sh`
```
Comment (ivy-lli) [inline README.md:18]: instead of 'code' clocks, you could also do links which leads directly to the correct files even with line numbers ;)

**Pull request merged**